migrations/versions/232b13d4e86a_.py

```diff
@@ -0,0 +1,32 @@
+"""empty message
+
+Revision ID: 232b13d4e86a
+Revises: 14e31b3a860b
+Create Date: 2019-06-17 02:51:42.186307
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '232b13d4e86a'
+down_revision = '14e31b3a860b'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('settings', sa.Column('google_recaptcha_secret', sa.String(), nullable=True))
+    op.add_column('settings', sa.Column('google_recaptcha_site', sa.String(), nullable=True))
+    op.add_column('settings', sa.Column('is_google_recaptcha_enabled', sa.Boolean(), nullable=False))
```

Comment:
This migration will fail

Reply:
@iamareebjamal i specified the default value above , its not showing in migration file , what should i do?

Reply:
See previous PRs where default has been supplied